org.lflang/src/org/lflang/generator/ReactionInstance.java

```diff
@@ -218,6 +227,14 @@ public ReactionInstance(
      */
     public TimeValue deadline = new TimeValue(TimeValue.MAX_LONG_DEADLINE, TimeUnit.NANO);
 
+    /**
+     * Sadly, we have no way to mark reaction "unordered" in the AST,
+     * so instead, we use a magic comment at the start of the reaction body.
+     * This is that magic comment.
+     */
+    public static String UNORDERED_REACTION_MARKER
+            = "// **** This reaction is unordered.";
```

Comment:
I think this is assuming that `//` is a comment indicator, which is fine for all of our target languages (except for Python, but it doesn't have a need for unordered reactions written in the Python language). Maybe this is a strong enough reason to introduce the `unordered` syntax?

Reply:
No, we should only every introduce new syntax if we can agree that this improves the utility of the programming language, not because of some internal compiler issue. So far, I have not seen a compelling use case or argument for such syntax. The only thing that would be needed to fix _this_ particular issue is to specify for each target what the single-line comment character sequence is and use that instead of a hard-coded string. But as you already pointed out, it isn't really an issue yet. My point is, despite this being a hack, it would generalize just fine. Another solution that generalizes well and is not a hack, is to put unordered reactions in separate reactors.

Reply:
This is a good idea. The 'magic string' could be made target-specific. How to do that?

Reply:
We can easily specify this in the `Target` class.

Reply:
It's not clear to me what should go in the Target class. For now, I've removed the comment prefix '// ' and use String.contains() rather than String.startsWith() (which is much less efficient because it has to search the whole string when there is no match). If someone wants to take over the use of Target, I would be grateful.

Reply:
You're hiding something in the comments, assuming that `//` is indeed recognized as the start of a comment. This is not true for all targets. If, in the `Target` enum you specify for each target what their single-line comment character sequence is, then you can use that instead so that the trick generalizes across targets. That said, the concern @Soroosh129 raised isn't really a concern right now, because the trick isn't being used in Python yet. I'm also fine deferring this.

Reply:
I agree. This shouldn't be a concern for a while. For this to be a concern, a new target language must come along that doesn't have `//` as a comment annotation and implement federated execution using that target language natively.

Reply:
I was more trying to clarify if others think we should add the `unordered` keyword or not based on the raised concern.